src/c/project-card.js

```diff
@@ -58,6 +58,9 @@ const projectCard = {
         const isFinished = project => _.contains(['successful', 'failed', 'waiting_funds'], project.state);
 
         const cardCopy = (project) => {
+            if (projectVM.isSubscription(project)) {
+                return m("img.product-label[src='https://daks2k3a4ib2z.cloudfront.net/5991cfb722e8860001b12d51/59cab35c8a10d00001601249_assinatura-label.png']")
```

Comment:
@vicnicius we should user our S3 account for image. I have uploaded it here: https://s3.amazonaws.com/cdn.catarse/assets/assinatura-label.png

Reply:
fixed! thx!